integrations/oci/metrics/metrics/src/test/java/io/helidon/integrations/oci/metrics/OciMetricsSupportTest.java

```diff
@@ -76,14 +79,21 @@ public boolean matches(MetricID metricID, Metric metric) {
     }
 
     @Test
-    public void testMetricUpdate() {
-        // mock monitoringClient.postMetricData()
+    public void testMetricUpdate() throws InterruptedException {
+        countDownLatch1 = new CountDownLatch(1);
+        countDownLatch2 = new CountDownLatch(1);
         doAnswer(invocationOnMock -> {
+            if (countDownLatch1.getCount() != 0) {
+                // Wait for 1st signal before we can proceed
+                countDownLatch1.await(10, java.util.concurrent.TimeUnit.SECONDS);
```

Comment:
We can simplify this to just
```java
countDownLatch1.await(...);
```
The `await` method will return immediately if the latch has already gone to 0; there is no need to check its value first.

Reply:
The goal really is for countDownLatch2.countdown() to be executed only when **countDownLatch1.getCount() == 0** which is the else part and should happen after the second call of this mocked method. If do something like this
```
countDownLatch1.await(10, java.util.concurrent.TimeUnit.SECONDS);
if (countDownLatch1.getCount == 0) {
    countDownLatch2.countDown();
}
```
then  `countDownLatch2.countDown()` will be executed right after  countDownLatch1.await(..) proceeds in the same call rather than it be executed on the next call?

Reply:
Modified to explicitly test 1st and 2nd metric update.